BREAKING.md

```diff
@@ -29,6 +30,9 @@ Remove deprecated `shipAssert` `debugAssert` `logException` `logGenericError` in
 ### fileOverwrittenInStorage
 Please use `DriverErrorType.fileOverwrittenInStorage` instead of `OdspErrorType.epochVersionMismatch`
 
+### absolutePath use in IFluidHandle is deprecated
+Please use `get` in IFluidHandle for routing instead of storing absolute paths.
```

Comment:
I think this is not sufficient description. We need to describe how to change existing workflows that rely on it.

Reply:
Rather than retrieving the absolute path, ostensibly to store, they should instead store the handle itself. To load they should retrieve the handle, and then call get on it to get the actual object.

Reply:
Changed it